index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that include transformations to the
```

Comment:
I'll repeat what I've said many times. 

Verifiable Credentials are what we created here in this working group. When we says "verifiable credential", we have ALWAYS meant W3C verifiable credentials.

Extending this term to anyone who makes a thing that converts to VCs is not appropriate nor will it help the ecosystem. 

I object to this PR

Reply:
@jandrieu if we removed the "transformations" text... would that work for you?

Reply:
I have attempted to address your concern in commit 01d179bcf7d2a54d6dc4cc0f5ea5c7b24b6ae60a. The language is a little dicey, so please help me improve it so it achieves what you want @jandrieu without triggering objections. :)